contributors/devel/development.md

```diff
@@ -242,54 +361,107 @@ To build binaries for a specific platform, add `KUBE_BUILD_PLATFORMS=<os>/<arch>
 make cross KUBE_BUILD_PLATFORMS=windows/amd64
 ```
 
-#### Install etcd
+## A Quick Start for Testing Kubernetes
 
-```sh
-cd $working_dir/kubernetes
+Kubernetes only merges pull requests when unit, integration, and e2e tests are
+passing, so it is important that your development environment can
+successfully run all tests. While this quick start will get you going,
+to really understand the testing infrastructure, read the
+[Testing Guide](sig-testing/testing.md) and check out the
+[SIG Architecture developer guide material](README.md#sig-testing).
 
-# Installs in ./third_party/etcd
-hack/install-etcd.sh
+Note that all of the commands in this section are run in your
+Kubernetes project directory at `$GOPATH/src/k8s.io/kubernetes/`
+unless otherwise specified.
 
-# Add to PATH
-echo export PATH="\$PATH:$working_dir/kubernetes/third_party/etcd" >> ~/.profile
```

Comment:
the `hack/install-etcd.sh` script tells you the export command, but it might still be worth mentioning in the etcd section that if they want to make the path permanent they should add it to their profile.

Reply:
Good call! I added a note to this effect.